src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -116,6 +121,15 @@
               optional = true)
     private Boolean overwriteExistingWorkspace = false;
 
+    @Argument(fullName = BATCHSIZE_ARG_NAME,
+              shortName = BATCHSIZE_ARG_NAME,
+              doc = "Batch size to group samples in during import. " +
+                    "This will minimize memory consumption, but might take longer to complete." +
+                    "Also, if batching creates too many fragments in GenomicsDB, run consolidation" +
+                    " to improve feature read time. Defaults to " + DEFAULT_ZERO_BATCH_SIZE,
+              optional = true)
+    private int batchSize = DEFAULT_ZERO_BATCH_SIZE;
```

Comment:
If the batchSize is greater than the number of samples, will the tool work / not blow up?